### PR TITLE
[#9536] Instructor Home Page E2E Test

### PR DESCRIPTION
**Pull request by jtankw3**
Part of #9536

**Outline of Solution**
- Some methods are copied from `InstructorFeedbackSessionsPageE2ETest`, as there are quite a few other methods across E2E test cases that have to be abstracted, they will be abstracted together in a separate PR.
- Add `InstructorHomePageE2ETest` which tests:
     - search for valid student
     - verify loaded data
     - verify response rate
     - copy session
     - publish results
     - send reminder email
     - resend results link
     - unpublish results
     - download results
     - soft delete session
     - archive course
     - delete course

### CODE REVIEW

File: src/e2e/java/teammates/e2e/cases/e2e/InstructorHomePageE2ETest.java
Comment (wkurniawan07) [inline src/e2e/java/teammates/e2e/cases/e2e/InstructorHomePageE2ETest.java:242]: We really should extract out this kind of operation

**Review by wkurniawan07 — APPROVED** [review]
LGTM, concise and yet effective as E2E test 💯

**Comment by madanalogy** [comment]
@jtankw3 you can update this PR's description to close #9676 as well. Should be good to go after you resolve the merge conflicts.

**Pull request merged**
